encodings/dict/benches/chunked_dict_array_builder.rs

```diff
@@ -0,0 +1,91 @@
+use divan::Bencher;
+use rand::distributions::{Distribution, Standard};
+use rand::prelude::StdRng;
+use rand::{Rng, SeedableRng};
+use vortex_array::array::{ChunkedArray, PrimitiveArray};
+use vortex_array::builders::builder_with_capacity;
+use vortex_array::{Array, IntoArray, IntoCanonical};
+use vortex_dict::DictArray;
+use vortex_dtype::NativePType;
+use vortex_error::{VortexExpect, VortexUnwrap};
+
+fn main() {
+    divan::main();
+}
+
+fn make_dict_primitive_chunks<T: NativePType, U: NativePType>(
```

Comment:
ah, so we have this logic also in other benchmarks in slightly different shape. Maybe we can have a function to generate random dictionaries behind a feature flag in main crate and then use `required-features` on the benchmark